src/Markdown.svelte

```diff
@@ -59,6 +62,25 @@
         });
       }
     }
+
+    // Replaces code blocks in the background with highlighted code.
+    const prefix = "language-";
+    const nodes = Array.from(document.body.querySelectorAll("pre code"));
+
+    for (const node of nodes) {
+      const className = Array.from(node.classList).find(name =>
+        name.startsWith(prefix),
+      );
+      if (!className) continue;
+      let tree: MaybeHighlighted;
+      (async () => {
```

Comment:
I guess we can also just make the anonymous function in onMount `async` to avoid the IIAFE here.

Reply:
1e545010a9ff419c4ecb5074f85b6e921604e78c